### PR TITLE
Support the latest pkg:http

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
Required for https://github.com/dart-lang/appengine/issues/88

File: pubspec.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 author: Dart Team <misc@dartlang.org>
 description: Dart gcloud APIs
 homepage: https://github.com/dart-lang/gcloud
+
 environment:
-  sdk: '>=2.0.0-dev.54.0 <3.0.0'
+  sdk: '>=2.0.0 <3.0.0'
+
 dependencies:
   _discoveryapis_commons: ^0.1.6+1
```
Comment (devoncarew) [inline pubspec.yaml]: Unrelated to this PR, but I see that `package:_discoveryapis_commons` is currently at `0.1.8+1`. Do you happen to know if this package requires a specific version, or if the range can be broadened?

Comment (kevmoo) [reply pubspec.yaml]: This constraint includes the latest _discoveryapis_commons

Comment (devoncarew) [reply pubspec.yaml]: Ah, yeah; I read that constraint wrong.

**Pull request merged**